### DISCUSSION TITLE
KTX: Collection utilities README.md typo?

### DISCUSSION THREAD

**Discussion by BitsAtPlayDev** [libktx/ktx#429] [General]
Hello, 

Under the **"Why?"** section of the KTX: Collection utilities README.md it reads: 

"Although standard Kotlin library features superb utilities, some of them can be used along with the libGDX collections simply because they do not implement the Collection interface."

Should it read:

"... some of them can _not_ be used along with ..." ?

**Comment by czyzby** [comment]
Hey. Yes, it's a typo - it should have been "cannot". :) You're welcome to submit a PR with a fix, otherwise I'll update it along with some other upcoming changes.

**Reply by czyzby** [reply]
Fixed, will commit soon to the `develop` branch.
